### PR TITLE
Small test data for SeqInspector pipeline

### PR DESCRIPTION
**Pull request by KarNair**
Relates to issue [#9](https://github.com/nf-core/seqinspector/issues/9) on [SeqInspector](https://github.com/nf-core/seqinspector)
This PR adds test datasets from Illumina instruments (MiSeq, and Novaseq6000) for the SeqInspector pipeline. Each datasets consists of:

- Input samplesheet
- Runfolder/Rundir containing only fastq files
- Tarballed runfolder/rundir which contains only InterOp files and L001 binaries

### CODE REVIEW

File: testdata/NovaSeq6000/samplesheet.csv [hunk 1]
```diff
@@ -0,0 +1,6 @@
+sample,lane,project,fastq_1,fastq_2,rundir
+Sample1,1,S1,https://github.com/KarNair/test-datasets/raw/seqinspector/testdata/NovaSeq6000/200624_A00834_0183_BHMTFYDRXX/Sample1_S1_L001_R1_001.fastq.gz,,
+SampleA,1,S2,https://github.com/KarNair/test-datasets/raw/seqinspector/testdata/NovaSeq6000/200624_A00834_0183_BHMTFYDRXX/SampleA_S2_L001_R1_001.fastq.gz,,
+Sample23,1,S2,https://github.com/KarNair/test-datasets/raw/seqinspector/testdata/NovaSeq6000/200624_A00834_0183_BHMTFYDRXX/Sample23_S3_L001_R1_001.fastq.gz,,
+sampletest,1,S4,https://github.com/KarNair/test-datasets/raw/seqinspector/testdata/NovaSeq6000/200624_A00834_0183_BHMTFYDRXX/sampletest_S4_L001_R1_001.fastq.gz,,
+Undetermined,1,,https://github.com/KarNair/test-datasets/raw/seqinspector/testdata/NovaSeq6000/200624_A00834_0183_BHMTFYDRXX/Undetermined_S0_L001_R1_001.fastq.gz,,
```
[the remark this answers is not in the document]
Comment (KarNair) [reply testdata/NovaSeq6000/samplesheet.csv]: Done!

File: testdata/MiSeq/220422_M11111_0222_000000000-K9H97.tar.gz
Comment (Aratz) [inline testdata/MiSeq/220422_M11111_0222_000000000-K9H97.tar.gz:1]: Shouldn't this be removed?

File: testdata/NovaSeq6000/200624_A00834_0183_BHMTFYDRXX.tar.gz
Comment (Aratz) [inline testdata/NovaSeq6000/200624_A00834_0183_BHMTFYDRXX.tar.gz:1]: Same comment here.

File: testdata/MiSeq/220422_M11111_0222_000000000-K9H97.tar.gz
Comment (KarNair) [reply testdata/MiSeq/220422_M11111_0222_000000000-K9H97.tar.gz:1]: We do not need the tar files containing interop right now. But we will need in the future for testing.

File: testdata/NovaSeq6000/200624_A00834_0183_BHMTFYDRXX.tar.gz
Comment (KarNair) [reply testdata/NovaSeq6000/200624_A00834_0183_BHMTFYDRXX.tar.gz:1]: We do not need the tar files containing interop right now. But we will need in the future for testing.

**Comment by kedhammar** [comment]
I added some minimal ONT test data, please let me know if there are any issues or if it should be handled separately @KarNair

**Comment by KarNair** [comment]
@kedhammar I think we do not need anything else at this point other than the fastq files and the samplesheet. I think it would be better if you remove other reports and so on (basically, anything that is not mentioned in the samplesheet can be tossed), to keep the repo clean.

**Review by jfy133 — COMMENTED** [review]
All of the FASTQ files are rather large, can you not make them smaller (e.g. subsampling reads)?

**Comment by kedhammar** [comment]
> All of the FASTQ files are rather large, can you not make them smaller (e.g. subsampling reads)?

[33fd71b](https://github.com/nf-core/test-datasets/pull/1163/commits/33fd71b1cf8914fe476fdfa261f8b43fe993051c) downsampled passed/failed reads to 50/10 respectively.

**Review by jfy133 — COMMENTED** [review]
fastq_fail samples have `fastq.gz` suffix, but are raw-text, is that to be expected?

And can you confirm the 8mb novaseq tar can't be made any smaller?

**Comment by kedhammar** [comment]
> fastq_fail samples have `fastq.gz` suffix, but are raw-text, is that to be expected?

Good catch @jfy133 , fixed in [81ad8f3](https://github.com/nf-core/test-datasets/pull/1163/commits/81ad8f3d11100e7da3bbd3868fd31a93c5fb477b)

**Comment by jfy133** [comment]
The tar question is the only outstanding thing from me now :)

**Comment by Aratz** [comment]
@jfy133 the bulk of the 8mb is made of the Illumina Interop Metric files (which give information about how the run went and can be use for quality control). Illumina provides a tool to analyze these files but no format specification, which makes it hard to reduce their size at all.

**Review by jfy133 — APPROVED** [review]

**Pull request merged**